about.php

```diff
@@ -0,0 +1,211 @@
+                <img src="images/desire-copy.jpg"> 
+			<h2 style="position:absolute; left:35%;bottom:1px">Yameogo Desire</h2>
+               <p class="dop-down6 show6"></p>
+            </li> 
+            <li id="li7">
+                <img src="images/gampine.jpg"> 
+				<h2 style="position:absolute; left:35%;bottom:1px"> Gampine Aristide</h2>
+               <p class="dop-down7 show7"> After his master's degree in English at the University of Ouagadougou, Burkina Faso, Aristides Gampine strengthened his abilities in innovative practices of teaching English with Endicott college in Switzerland. he has taught at various international schools and at the American language center. Gampine is pleased that young Burkinabe have the opportunity to take up the challenge of studying entrepreneurship in English at BIT.</p>
+            </li>
+        </ul>
+    </div>
+ <h1>Ours team</h1>
+<p style="text-align:center;width:80%;font-family:Adobe Caslon Pro Bold;margin:auto;opacity:0.6">All of us in the team are studying at Burkina Institute of Technology. To be honest, for us computer science is in the center of our lives we really like it. Knowing that it is very important for every country in the world, our dream is to discover the mystery that is in the back side of the computers. We appeal all the youth to have passion for it. learn it and you will be impressed about what you will find as result. You can also join our team if you love also computer Science and to understand a lot about it <a href="contact.php"><i>Join us</i></a>. For those who are Expert in the domain and want to help us, you are welcome. just contact us.  </p>
+    <script src="js/js.js" type="text/javascript"></script>
+    <div id="owner">
+       
+        <ul>
+    <div class="block-2 ">
+            
+              <div class="front">
+                <img src="images/Dolores.jpg">
+                  <style>
+                      .front img{
+                          position: absolute;
+                          width: 100%;
+                          height: 100%;
+                      }
+                  </style>
+                <div class="box">
+                  <h2>Neya Dolores</h2>
+                </div>
+              </div>
+              <div class="back">
+                
+                <blockquote>
+                  <p>&ldquo;My name is Neya Dolores. I'm 19 years old and I'm a student of a BIT. I am a big fan of technology. I like programming and management. We invite you to get involved in this field.  I have knowledge in English, French and German. Programming languages such as C, JavaScript and PHP are not unknown to me. Thank you very much.&rdquo;</p>
+                </blockquote>
+              </div>
+            </div>
+                      <li class="block-2 ">
+            <div class="flipper">
+              <div class="front">
+                   <img src="images/jere.jpg">
+                  <style>
+                      .front img{

```

Comment:
As you defined it similar above, I'd really put this into a separate file and have a common class for it. 

It might look like that 
```
.owner-image {
      position: absolute;
      width: 100%;
      height: 100%;
}

.owner-image__opacity {
      opacity: 0.7;
}
```

After that you have something like this in the HTML file: 

```
<img class="owner-image" src="images/Dolores.jpg">
...
<img class="owner-image owner-image__opacity" src="images/jere.jpg">
```